### ISSUE TITLE
Internal server error when fetching key pairs

### ISSUE THREAD

**Issue by hackdna** [galaxyproject/cloudlaunch#64]
Commit afbae38 running at http://cloudman.aws.stemcellcommons.org/launch.

### Steps to reproduce ###
Enter access and secret keys, choose a saved cluster then fetch the key pairs.

### Observed results ###
The key pair drop down is disabled and filled with some Javascript code.
Browser console:
```
jquery-2.0.2.min.js:6 POST http://cloudman.aws.stemcellcommons.org/get-key-pairs 500 (INTERNAL SERVER ERROR)
```
/srv/cloudlaunch/cloudlaunch/access.log:
```
127.0.0.1 - - [30/Aug/2016:18:24:45 -0400] "POST /get-key-pairs HTTP/1.0" 500 365 "http://cloudman.aws.stemcellcommons.org/launch" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
```
However, there is nothing in /srv/cloudlaunch/cloudlaunch/error.log or in /var/log/nginx/error.log. Is there a way to see the actual server error (traceback)?

**Comment by afgane** [comment]
If `DEBUG` is set to `False` in `settings.py` (or `settings_local.py`), the server error will get emailed to the users listed as `ADMINS` in that same file, assuming the server has the ability to send mail. You could set `DEBUG` to `True` briefly and you'd then get the error in the server logs. 

launch.usegalaxy.org server is not seeing the same issue so it's probably got to do with the server setup.

**Comment by hackdna** [comment]
Thanks for getting back quickly. This EC2 instance is not configured to send email, so I've added `EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"` in addition to `DEBUG = True` to `settings_local.py` but the error was still not getting logged in either `/srv/cloudlaunch/cloudlaunch/error.log`, `/srv/cloudlaunch/cloudlaunch/supervisor.log` or `/var/log/nginx/error.log`.

After that I switched the config back to the original and now there is no more error when fetching key pairs, so I'm at a loss regarding the cause.

The only other change I've made was to adjust the system time using ntp service (it was ahead by about 5 minutes) but I'm not sure how this could be related. In any case, looking forward to the all new CloudLaunch :)
